Documentation/reference/os/iob.rst

```diff
@@ -164,6 +164,7 @@ Public Function Prototypes
   - :c:func:`iob_remove_queue()`
   - :c:func:`iob_peek_queue()`
   - :c:func:`iob_free_queue()`
+  - :c:func:`iob_free_qentry_from_queue()`
```

Comment:
@anchao and @antmerlino how about name the new function iob_free_queue_qentry?

Reply:
@xiaoxiang781216 works for me